app/assets/javascripts/trim_link.js

```diff
@@ -3,9 +3,47 @@
 
 var trimLink = {};
 
+
 // Prepare the callback on trim upload controls
 trimLink.trimFileUpload = function() {
   'use strict';
+  var $fileField = $('input[type=file]');
+
+  if ($('#pq-details').length) {
```

Comment:
add a comment that this executes only on the details pages. More comments elsewhere welcome too